### PR TITLE
Remove everything that isn't important to the process

### PR DESCRIPTION
**Pull request by joewalker**
@mykmelez I think you've seen this before. It's not improving sitting on my hard drive...

### CODE REVIEW

**Comment by ncalexan** [comment]
I'm fine with this.  It's obviously a lot more stripped down and bare bones, but that's probably reasonable for a document that grew out of my (subjective) frustration with a sub-par process and therefore needed to give a lot of colour to a better future, but is now functioning as a template for other parts of the organization.

File: text/0006-architecture-review-process.md [hunk 1]
```diff
@@ -40,15 +40,16 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - how the proposal handles these scenarios
       - what review and discussion of the proposed architecture has already happened (and with whom)
     * A competitive analysis suggesting alternatives, costs, and opportunities
+    * Input by representatives from the affected groups. The team is expected to listen to this input and update the review packet based on the input, but it is likely that not all the questions or comments can be answered before the review.
```
Comment (mykmelez) [inline text/0006-architecture-review-process.md]: I worry that expecting the team to "listen to this input" will rub people the wrong way, and it seems ancillary to what seems to be the main point, which is that the team is *not* required to answer all questions and address all comments before the review. I would rewrite this sentence to focus on that main point, something like:

* Input by representatives from the affected groups. The team may update the review packet based on this input but is not required to answer all questions and address all comments before the review.

File: text/0006-architecture-review-process.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
...
 
-2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key issues so the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the review are likely to be more significant. The Review Packet includes:
-    * A lay summary of the problem space which is focused on defining a shared language and identifying the key forces behind the problem
+2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The Review Packet includes:
+    * A lay summary of the problem space which is focused on defining a shared language and identifying the key forces behind the problem.
```
Comment (mykmelez) [inline text/0006-architecture-review-process.md]: Since you're changing this line anyway, here's a recommendation to simplify it (for clarity):

* A lay summary of the problem space that defines a shared language and identifies the key forces behind the problem.

File: text/0006-architecture-review-process.md [hunk 3]
```diff
@@ -40,15 +40,16 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - how the proposal handles these scenarios
       - what review and discussion of the proposed architecture has already happened (and with whom)
     * A competitive analysis suggesting alternatives, costs, and opportunities
+    * Input by representatives from the affected groups. The team is expected to listen to this input and update the review packet based on the input, but it is likely that not all the questions or comments can be answered before the review.
 
-3. The reviewer creates a list of questions to be discussed during the review. These questions should be:
+3. The reviewer creates a list of questions to be discussed during the review, possibly from the comments made in step 2. These questions should be:
     * Open-ended
```
Comment (mykmelez) [inline text/0006-architecture-review-process.md]: You aren't touching this line, but it occurs to me that I'm uncertain why this matters. Is it that open-ended questions are more likely to foster productive analysis and discussion? I wonder if there's more we can say here about the kinds of questions that are helpful for reviews.

File: text/0006-architecture-review-process.md [hunk 4]
```diff
@@ -57,28 +58,20 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - A step back to discuss if the proposal is right in the wider context
     * A lightweight review might not need the input of others, and ultimately, might not even need a specific review meeting.
 
-The review meeting should be a discussion of the issues, but should avoid feeling pressured to make a decision. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review. The meeting may decide to alter the questions and undergo a subsequent review. We should be very open about conversations that happen after the review meeting.
+The review meeting should be a discussion of the issues, but should avoid feeling pressured to make a decision. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review.
```
Comment (mykmelez) [inline text/0006-architecture-review-process.md]: I know this is existing verbiage, but here too I worry about how potential reviewers will react to being told how to feel (i.e. that they "should avoid feeling pressured"). I would reword to emphasize just that reviewers don't need to make a decision in the meeting (without reference to what they may or may not feel), something like:

> The review meeting should be a discussion of the issues. Reviewers don't need to make a decision at the meeting. The goal is to understand the issues raised by the question(s) and the background from the review packet, and to add to it wisdom from the people at the review.

File: text/0006-architecture-review-process.md [hunk 5]
```diff
@@ -57,28 +58,20 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
...
 
-For particularly small reviews it is possible to merge a roadmap review and design review into a single review, in which case the question is both 'should we do this', and 'should we do it this way'. It is possible to conduct combined reviews where the total time for the review for all participants is 1 working day.
+A more negative review may decide to alter the questions, undergo a subsequent review or drop the subject entirely. It is a goal of the process that we discover fatal flaws as early on in the process as possible.
 
-For both a Roadmap Review and a Design Review the goal is to hear perspectives that will lead to a better definition of a problem or solution, and better judgment about whether it should be solved or how to solve it.
+The ideal review process scales well. The same basic system should work for a quick 2 person decision over the best way to design a certain feature, and for a critical organization-wide decision about a path to take. The term “team” is used above although we strongly recommend design reviews for smaller questions. If you feel yourself wondering if some design is best, it should be easy to perform a review.
```
Comment (mykmelez) [inline text/0006-architecture-review-process.md]: This paragraph is incongruous because it's meta—a commentary on what the ideal review process would be—whereas the rest of the section is a description of an actual review process. I wonder if there's a better place for it, perhaps at the top of the document, as part of a statement of intent for the doc.

**Review by mykmelez — CHANGES_REQUESTED** [review]
[blind-review](https://github.com/zombie/blind-reviews "Review performed in blind mode")

**Comment by joewalker** [comment]
Total tangent: It seems strange that @mykmelez was able to [commit](https://github.com/joewalker/firefox-browser-architecture/commit/54982852f76b42dc351e80a8b5b1d19a653ce738) to my fork of this repo despite not being listed as a collaborator in settings. And yet when I asked Brendan for a comments on a [PR](https://github.com/joewalker/firefox-browser-architecture/pull/3) he wasn't able to give a review (in the GitHub sense).

This may make sense from some angle, but it seems relevant to [the discussion](https://docs.google.com/document/d/1h_h7Armxv6NQMDyAdIE1TyILc7nnYcBr1cK7FYthJy4/edit?disco=AAAABtQJG_o)

File: text/0006-architecture-review-process.md (hunk 1 above)
Comment (joewalker) [reply text/0006-architecture-review-process.md]: Fixed in 921a0c87

File: text/0006-architecture-review-process.md (hunk 2 above)
Comment (joewalker) [reply text/0006-architecture-review-process.md]: Fixed in: 219e8bf4

File: text/0006-architecture-review-process.md (hunk 3 above)
Comment (joewalker) [reply text/0006-architecture-review-process.md]: Yes, that was the point of stipulating open-ended, and we should specify the end not the means. 'Designed to foster productive analysis and discussion' works for me in its place.

Comment (joewalker) [reply text/0006-architecture-review-process.md]: Fixed in 782db537

File: text/0006-architecture-review-process.md (hunk 4 above)
Comment (joewalker) [reply text/0006-architecture-review-process.md]: Fixed in 3f828ffc

File: text/0006-architecture-review-process.md (hunk 5 above)
Comment (joewalker) [reply text/0006-architecture-review-process.md]: I've tried adding a new sub-heading because the next paragraph depends on this one and I didn't want to move them both to the top. You may need to see the change in context for it to make sense.

Comment (joewalker) [reply text/0006-architecture-review-process.md]: Updated in 7e56f0df

**Review by mykmelez — APPROVED** [review]

**Comment by mykmelez** [comment]
> Total tangent: It seems strange that @mykmelez was able to commit to my fork of this repo despite not being listed as a collaborator in settings. And yet when I asked Brendan for a comments on a PR he wasn't able to give a review (in the GitHub sense).
> 
> This may make sense from some angle, but it seems relevant to the discussion

Indeed, this is strange.

Perhaps GitHub reasons that since I can make any change to the upstream repo (mozilla/firefox-browser-architecture), I should be able to make any change to a branch that you want to merge into the upstream repo, even if the branch is in a repo (joewalker/firefox-browser-architecture) where I can't normally make any changes.

Or perhaps it's something to do with my role in the [browser-arch team](https://github.com/orgs/mozilla/teams/browser-arch) (I'm its "maintainer") or the [mozilla organization](https://github.com/mozilla) (I'm an "owner"). In any case, it's worth noting in the document you referenced, and I've done so via a footnote in that document.

**Comment by mykmelez** [comment]
Since I've approved these changes formally (via the GitHub review feature), and @ncalexan has done so informally (via a comment), I'll merge them.
